.github/workflows/markdown_to_pdf.yml

```diff
@@ -0,0 +1,31 @@
+name: Docs to PDF
+on:
+  push:
+    branches:
+      - master
+    # Paths can be used to only trigger actions when you have edited certain files, such as a file within the /docs directory
```

Comment:
Can you change this comment? We have no `/docs` directory, our sources are `/src`.

Reply:
Yes, sorry about that. I have tested on a personal repo using /docs directory. I really apologize, @schneiderfelipe

Reply:
That's OK. It's just for our own organization. Who knows what that will mean five years from now 😆